runtime/Cargo.toml

```diff
@@ -67,6 +67,7 @@ features = ["full"]
 
 [dev-dependencies]
 # For storage interoperability tests only.
+hex = "0.4"
```

Comment:
This should be moved below as the above comment applies to the `jsonrpc` dependency.